src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -225,7 +214,7 @@
   </PropertyGroup>
   
   <Target Name="CreateVersionInfoFile"
-          DependsOnTargets="GetLatestCommitHash"
+          DependsOnTargets="CreateVersionFileDuringBuild"
           BeforeTargets="BuildAllProjects"
           Inputs="$(LatestCommit)"
```

Comment:
It may only do something in the case where a file happens to have the same name as the commit hash. I can remove it.